resources/public/api/conf/1.0/application.raml

```diff
@@ -14,7 +14,7 @@ documentation:
  - title: Errors
    content: !include docs/errors.md
 
-mediaType: application/xml
```

Comment:
quickly, why is this just media type and not the hmrc one?

Reply:
Going by the RAML producers guide - this specifies the types that are accepted into the API but not the version.